scipy/stats/morestats.py

```diff
@@ -206,15 +204,42 @@ def kstat(data, n=2):
 
     http://mathworld.wolfram.com/Cumulant.html
 
+    Examples
+    --------
+    >>> from scipy import stats
+    >>> rndm = np.random.RandomState(1234)
+
+    As sample size increases, n-th moment and n-th k-statistic converge to the
+    same number (although they aren't identical)
+
+    >>> for n in [2, 3, 4, 5, 6, 7]:
+    ...     x = rndm.normal(size=10**n)
+    ...     m, k = stats.moment(x, 3), stats.kstat(x, 3)
+    ...     print m, k, m-k
+    -0.63144956970249455, -0.65084474304524276, 0.019395173342748206
+    0.028228612555103411, 0.028313496417362614, -8.4883862259202414e-05
+    -0.045424541104455429, -0.04543817164718604, 1.3630542730611039e-05
+    7.5338208014728139e-05, 7.5340468213851364e-05, -2.2601991232247673e-09
+    0.0016645501352486995, 0.0016645551289107396, -4.9936620400824888e-09
+    -2.8779648300317205e-06, -2.8779656933968186e-06, 8.6336509815359533e-13
```

Comment:
Nitpick: This, I think, will look nicer with some formatting of numbers: I'd replace the print with e.g. `print "%.3g  %.3g  %.3g" %(m, k, m-k)` or equivalent.